cli/src/pcluster/config/cluster_config.py

```diff
@@ -953,6 +953,7 @@ def __init__(
         additional_packages: AdditionalPackages = None,
         tags: List[Tag] = None,
         iam: ClusterIam = None,
+        config_region: str = None,
```

Comment:
I'd call it just `region`

Reply:
Calling it `region` breaks unit tests because the output of `cluster_schema.dump(cluster)` in `test_cluster_schema` will contains region information getting from the `property` `region` in `cluster_config`.
We can talk if this reply is confusing